spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/ConvertModel.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.utils
+
+import com.intel.analytics.bigdl.nn.Module
+import com.intel.analytics.bigdl.utils.caffe.CaffeLoader
+import scopt.OptionParser
+
+object ConvertModel {
```

Comment:
do we support tensorflow?

Reply:
Only tested bigdl, caffe, torch. Haven't tested tensorflow.